homeassistant/components/awattar/coordinator.py

```diff
@@ -0,0 +1,58 @@
+"""The Coordinator for aWATTar."""
+from __future__ import annotations
+
+from datetime import timedelta
+from typing import NamedTuple
+
+from awattar import AsyncAwattarClient, AwattarConnectionError, MarketItem
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+from homeassistant.util import dt as dt_util
+
+from .const import CONF_COUNTRY, COUNTRY_TO_CODE, DOMAIN, LOGGER, SCAN_INTERVAL
```

Comment:
I don't see that `SCAN_INTERVAL` is used anywhere else, so keep the definition in this module instead of const

Reply:
done.